internal/service/alertconfiguration/data_source_alert_configuration.go

```diff
@@ -1,22 +1,24 @@
-package mongodbatlas
+package alertconfiguration
 
 import (
 	"context"
 	"fmt"
+	"sort"
 
 	"github.com/hashicorp/hcl/v2/hclwrite"
 	"github.com/hashicorp/terraform-plugin-framework-validators/stringvalidator"
 	"github.com/hashicorp/terraform-plugin-framework/datasource"
 	"github.com/hashicorp/terraform-plugin-framework/datasource/schema"
 	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
 	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/mongodb/terraform-provider-mongodbatlas/internal/config"
 	"github.com/mongodb/terraform-provider-mongodbatlas/mongodbatlas/util"
 	"github.com/zclconf/go-cty/cty"
 	"go.mongodb.org/atlas-sdk/v20231115001/admin"
 )
 
-var _ datasource.DataSource = &AlertConfigurationDS{}
-var _ datasource.DataSourceWithConfigure = &AlertConfigurationDS{}
+var _ datasource.DataSource = &alertConfigurationDS{}
```

Comment:
making it private as it's only needed inside alertconfiguration package cc @marcosuma